runners/apex/pom.xml

```diff
@@ -38,6 +38,7 @@
     <skipIntegrationTests>true</skipIntegrationTests>
     <!-- memory limit for embedded cluster -->
     <surefire.args>-Xmx2048m</surefire.args>
+    <beam-top>../..</beam-top>
```

Comment:
you have commented that this may all be unnecessary if we're not using the `site` lifecycle.